Valeriia/Movies_database_project/src/components/MovieItem/MovieItem.tsx

```diff
@@ -13,19 +14,31 @@ const MovieItem = ({
   title,
   release_date,
   genres,
-}: IMovieItemProps) => (
-  <div className="item">
-    <div className="item__poster">
-      <img src={poster_path} alt="movie" className="item__img" />
-    </div>
-    <div className="item__info">
-      <span>{title}</span>
-      <span className="item__date">{release_date.substr(0, 4)}</span>
-    </div>
-    <div className="item__genre">
-      <span>{genres.join(", ")}</span>
+}: IMovieItemProps) => {
+  const handlerImage = (e: React.SyntheticEvent<HTMLImageElement>) => {
+    const target = e.target as HTMLImageElement;
+    target.src = placeholder;
+  };
+
+  return (
+    <div className="item">
+      <div className="item__poster">
+        <img
+          src={poster_path}
+          onError={(e) => handlerImage(e)}
+          alt="movie"
+          className="item__img"
+        />
```

Comment:
You are adding handlerImage twice. Seems like better to make a reusable component Image where you add onError handler and show placeholder, so you won't need to write handlerImage all the time.